# Chapter 4 Coding Exercises

## Algorithm Worthbench 

Question 3: 
Write a loop the displays the following set of numbers: 
0, 10, 20, 30, 40, 50 ... 1000

In [6]:
#Setting up Range for set of Numbers
for num in range(0, 1001, 10):
    #Displaying set of numbers 
    print(num)

0
10
20
30
40
50
60
70
80
90
100
110
120
130
140
150
160
170
180
190
200
210
220
230
240
250
260
270
280
290
300
310
320
330
340
350
360
370
380
390
400
410
420
430
440
450
460
470
480
490
500
510
520
530
540
550
560
570
580
590
600
610
620
630
640
650
660
670
680
690
700
710
720
730
740
750
760
770
780
790
800
810
820
830
840
850
860
870
880
890
900
910
920
930
940
950
960
970
980
990
1000


Question 2: Write a while loop that asks the user to enter two numbers. The numbers should be added and the sum displayed. The loop should ask the user if he or she wishes to preform the operation again. If so

## Programming Exercises 

Question 1: Bug Colector - a bug collector collects bugs every day for five days. Write a program that keeps a running total of the number of bugs collected during the five days. The loop should ask for the number of bugs collected each day, and when the loop is finished, the program should display the total number of bugs collected. 

Question 4: Distance Traveled - The distance a vehicle travels can be calculated as follows: 

distance = speed * time 

For example, if a train travels 40 miles per hour for three hours, the distance traveled is 120 miles. Write a program that asks the user for the speed of a vehicle ( in miles per hour) and the nnumber of hours it has traveled. It shouldthen use a loop to display the distance the vehicle has traveled for each hour of the time period. 

Question 12: Calculating the Factorial of a Number - Write a program that lets the user enter a nonnegative integer then uses a loop to calculate the factorial of that number. Display the factorial. 

Question 15: Write a program that uses nested loops to draw the pattern listed in the book. 